docs/python_docs/python/tutorials/performance/index.rst

```diff
@@ -111,7 +111,7 @@ Distributed Training
 
    .. card::
       :title: Data Parallelism in MXNet
-      :link: /api/faq/multi_devices.html
+      :link: /api/faq/multi_device
```

Comment:
what does this link to?

Reply:
This links to this page: https://mxnet.incubator.apache.org/api/faq/multi_device